cocos/audio/win32/AudioCache.cpp

```diff
@@ -224,9 +224,8 @@ void AudioCache::readDataTask()
     if (_pcmDataSize <= PCMDATA_CACHEMAXSIZE)
     {
         _pcmData = malloc(_pcmDataSize);
-        auto alError = alGetError();
```

Comment:
> To generate a set of buffers for use, use alGetError to reset the error state, call alGenBuffers to generate the number of buffers desired, and then use alGetError again to detect if an error was generated.
https://www.openal.org/documentation/OpenAL_Programmers_Guide.pdf  (page 9)

Please add a `alGetError();` back in front of `alGenBuffers(....);`, just don't store the result.
Same goes for the change in AudioPlayer.cpp